app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Expected format: 'mv source destination'"
+        )
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if not os.path.exists(source_path):
+        raise FileNotFoundError(f"Source file {source_path} does not exist")
+
+    if destination_path.endswith("/"):
+        os.makedirs(destination_path, exist_ok=True)
+        destination_path = os.path.join(
+            destination_path, os.path.basename(source_path)
```

Comment:
Consider using `os.path.join` for path concatenation, which is already correctly used here. This ensures the code is cross-platform.